themes/ansible-community/sass/_homepage-catalyst-band.scss

```diff
@@ -0,0 +1,60 @@
+.homepage-catalyst-band {
+  background-color: $white;
+}
+
+.catalyst-card {
+  display: flex;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  padding: 2rem;
+  border-width: 1px;
+  border-radius: 5px;
+  background-color: $white;
+  justify-content: stretch;
+}
+
+.catalyst-content {
+  display: flex;
+  flex-direction: column;
+  padding-right: 2rem;
+  justify-content: space-between;
+
+  h3 {
+    font-size: $font-lg;
+    font-weight: 600;
+    text-transform: none;
```

Comment:
Move the font color from the inline style to here.

Reply:
If you can implement that change for me I would appreciate it. My focus is on other activities today.

Reply:
Nevermind.  Change implemented.